### PR TITLE
Fix build errors in Wasp 0.10.0

### PR DESCRIPTION
**Pull request by sodic**

### CODE REVIEW

File: waspc/src/Wasp/Generator/ServerGenerator.hs [hunk 1]
```diff
@@ -177,7 +177,7 @@ npmDepsForWasp spec =
             ("@types/express", "^4.17.13"),
             ("@types/express-serve-static-core", "^4.17.13"),
             ("@types/node", "^18.11.9"),
-            ("@tsconfig/node" ++ show (major latestMajorNodeVersion), "^1.0.1")
+            ("@tsconfig/node" ++ show (major latestMajorNodeVersion), "1.0.1")
```
Comment (sodic) [inline waspc/src/Wasp/Generator/ServerGenerator.hs:180]: Temporarily fixes this issue: https://github.com/tsconfig/bases/issues/173

File: waspc/data/Generator/templates/react-app/src/auth/forms/Auth.tsx [hunk 2]
```diff
@@ -529,7 +529,7 @@ const VerifyEmailForm = (
 
   async function submitForm() {
     if (!token) {
-      setErrorMessage('The token is missing from the URL. Please check the link you received in your email.')
+      setErrorMessage({ title: 'The token is missing from the URL. Please check the link you received in your email.' })
```
Comment (sodic) [inline waspc/data/Generator/templates/react-app/src/auth/forms/Auth.tsx:532]: Fixes this error:
image

**Review by Martinsos — APPROVED** [review]
LGTM! Why temporary fix -> what would be the permanent fix?

**Review by shayneczyzewski — APPROVED** [review]
Tested and it works for me, thanks @sodic 🚀

**Pull request merged**